src/components/sunAndMoon.svelte

```diff
@@ -1,40 +1,39 @@
 <script lang="ts">
-	import { FadeDuration } from '$lib/song';
 	import { gameStore } from '$lib/stores/game';
+	import { playerStore } from '$lib/stores/player';
 
 	export let disabled: boolean;
-	export let handleStateChange: () => void;
 
-	let running: boolean;
+	$: running = $playerStore.fading;
```

Comment:
Maybe use the store directly and spare the extra variable

Reply:
done